### PR TITLE
Display the license HTML instead of the plain text

### PR DESCRIPTION
**Pull request by joelverhagen**
Mimic the display of license detail that the SPDX website uses Instead of scrubbing license placeholders, they are displayed in a different color.

I chose to display:

- License name, in addition to the identifier
- Standard license header (if available)
- Notes (if available)
- A link to the SPDX site
- A notice at the bottom linking to docs and saying when the data was refreshed.

Fix https://github.com/NuGet/NuGetGallery/issues/9344

### Before

image

### After

image

### Before

image

### After

image

### Before

image

### After

image

### CODE REVIEW

**Comment by zhhyu** [comment]
Looking...

**Review by zhhyu — APPROVED** [review]

File: src/NuGet.Licenses/Content/Site.css [hunk 1]
```diff
@@ -24,4 +24,12 @@ pre {
 
 #license-expression > a {
     white-space: nowrap;
-}
\ No newline at end of file
+}
+
+.optional-license-text {
+    color: #0000cc;
```
Comment (agr) [inline src/NuGet.Licenses/Content/Site.css:30]: Why color value here is lowercase and couple of lines below is upper-case? :)

Comment (agr) [reply src/NuGet.Licenses/Content/Site.css:30]: I assume it passes accessibility checks.

**Review by agr — APPROVED** [review]

File: src/NuGet.Licenses/Content/Site.css (hunk 1 above)
Comment (joelverhagen) [reply src/NuGet.Licenses/Content/Site.css:30]: I copied it verbatim from the SPDX site.

a11y passes. Thanks for the reminder.

**Pull request merged**